app/views/govuk_publishing_components/components/docs/taxonomy_list.yml

```diff
@@ -0,0 +1,93 @@
+name: Taxonomy list
+description: Wraps the highlight box, document list and image card components in one list. This is commonly used on topic pages for links to tagged content.
+body: |
+  The [highlight box](/component-guide/highlight_boxes), [document list](/component-guide/document_list) and [image card](/component-guide/image_card) components are standalone components.
+
+  However, there are some use cases where we want to use both components to render one list of links. This is not possible when using the two components separately.
+  This component wraps these two components in one list, and applies relevant styling to get them to display appropriately together.
+accessibility_criteria: |
+  The taxonomy list should:
+
+  - Wrap all links in one list
+  - Should not be used to display links that don't belong together in one list
+shared_accessibility_criteria:
+  - link
+examples:
+  default:
+    data:
+      highlight_box:
+        inverse: true
+        items:
+        - link:
+            text: If your child is taken into care
+            path: /child-into-care
+            description: Information on what to do if your child is taken into care
+          metadata:
+            document_type: Detailed guide
+        - link:
+            text: High needs funding
+            path: /high-needs-funding
+          metadata:
+            document_type: Guide
+            public_updated_at: 2016-06-27 10:29:44
```

Comment:
Just for testing the layout I'd suggest adding another two `link` entries in here - so the displayed example has four highlight boxes.